src/main/java/core/basesyntax/model/FruitDto.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.model;
+
+import com.opencsv.bean.CsvBindByName;
+import com.opencsv.bean.CsvCustomBindByName;
+import java.util.Objects;
+
+public class FruitDto {
```

Comment:
```suggestion
public class TransactionDto/StoreOperationDto {
```
i believe this naming will better represent fields that we have in this class...